client/post-editor/editor-html-toolbar/index.jsx

```diff
@@ -0,0 +1,309 @@
+		const datetime = this.props.moment().format();
+		this.insertHtmlTag( { name: 'del', attributes: { datetime } } );
+	}
+
+	onClickInsert = () => {
+		const datetime = this.props.moment().format();
+		this.insertHtmlTag( { name: 'ins', attributes: { datetime } } );
+	}
+
+	onClickImage = attributes => {
+		this.insertHtmlTagSelfClosed( { name: 'img', attributes } );
+	}
+
+	onClickUnorderedList = () => {
+		this.insertHtmlTag( { name: 'ul', options: { newLine: true } } );
+	}
+
+	onClickOrderedList = () => {
+		this.insertHtmlTag( { name: 'ol', options: { newLine: true } } );
+	}
+
+	onClickListItem = () => {
+		this.insertHtmlTag( { name: 'li' } );
+	}
+
+	onClickCode = () => {
+		this.insertHtmlTag( { name: 'code' } );
+	}
+
+	onClickMore = () => {
+		this.insertCustomContent( '<!--more-->' );
+	}
+
+	onClickCloseTags = () => {
+		const closedTags = reduce(
+			this.state.openTags,
+			( tags, openTag ) => this.closeHtmlTag( { name: openTag } ) + tags,
+			''
+		);
+		this.insertCustomContent( closedTags );
+		this.setState( { openTags: [] } );
+	}
+
+	openImageDialog = () => {
+		this.setState( { showImageDialog: true } );
+	}
+
+	closeImageDialog = () => {
+		this.setState( { showImageDialog: false } );
+	}
+
+	openLinkDialog = () => {
+		const { inner: selectedText } = this.splitEditorContent();
+		this.setState( {
+			selectedText,
+			showLinkDialog: true,
+		} );
+	}
+
+	closeLinkDialog = () => {
+		this.setState( { showLinkDialog: false } );
+	}
+
+	tagLabel( tag, label ) {
+		const { openTags } = this.state;
+		return -1 === openTags.indexOf( tag ) ? label : `/${ label }`;
+	}
+
+	render() {
+		if ( ! config.isEnabled( 'post-editor/html-toolbar' ) ) {
+			return null;
+		}
+		const { translate } = this.props;
+		const buttons = {
+			bold: {
+				label: this.tagLabel( 'strong', 'b' ),
+				onClick: this.onClickBold,
+			},
+			italic: {
+				label: this.tagLabel( 'em', 'i' ),
+				onClick: this.onClickItalic,
+			},
+			link: {
+				label: 'link',
+				onClick: this.openLinkDialog,
+			},
+			quote: {
+				label: this.tagLabel( 'blockquote', 'b-quote' ),
+				onClick: this.onClickQuote,
+			},
+			del: {
+				label: this.tagLabel( 'del', 'del' ),
+				onClick: this.onClickDelete,
+			},
+			ins: {
+				label: this.tagLabel( 'ins', 'ins' ),
+				onClick: this.onClickInsert,
+			},
+			image: {
+				label: 'img',
+				onClick: this.openImageDialog,
+			},
+			unorderedList: {
+				label: this.tagLabel( 'ul', 'ul' ),
+				onClick: this.onClickUnorderedList,
+			},
+			orderedList: {
+				label: this.tagLabel( 'ol', 'ol' ),
+				onClick: this.onClickOrderedList,
+			},
+			listItem: {
+				label: this.tagLabel( 'li', 'li' ),
+				onClick: this.onClickListItem,
+			},
+			code: {
+				label: this.tagLabel( 'code', 'code' ),
+				onClick: this.onClickCode,
+			},
+			more: {
+				label: 'more',
+				onClick: this.onClickMore,
+			},
+			closeTags: {
+				disabled: ! this.state.openTags.length,
+				label: translate( 'Close Tags' ),
```

Comment:
@yoavf This is a weird one.
I don't mind replacing it with the lowercase version as suggested here, since it will be displayed lowercase anyway.
But, I went and checked the suggestion. It's in the localisation list of quick tags on WP Admin, and apparently it's the only non capitalised one.
What is the best thing to do here? Replace it with the existing string or keep mine?